backend/src/auth/guards/jwt-auth.guard.ts

```diff
@@ -0,0 +1,5 @@
+import { Injectable } from "@nestjs/common";
+import { AuthGuard } from "@nestjs/passport";
+
+@Injectable()
+export class JwtAuthGuard extends AuthGuard("jwt") {}
```

Comment:
Consider more avanced guards. I don't know exact project goals and how api should works. But for now e.g. each user could get information about other users by `user` endpoint. But meybe it's how it should work.

Reply:
That's also validate now, i don't push it yet :)